### DISCUSSION TITLE
[Feature Request] Maximum amount of backups to keep.

### DISCUSSION THREAD

**Discussion by JohanVlugt** [mbround18/valheim-docker#740] [Ideas]
Hello,

It would be good to have an ENV variable where you can set the maximum amount of backups to keep, for example 100 backups.

My backups are set to only backup when players are online. And with the maximum days to keep  it can happen that all backups get deleted if I don't play for a some time.

The goal is to have only relevant backups available when users were online. With the current options I can choose to keep the backups for an x amount of days which deleted timed out backups or to keep them forever which bloats the disk.

With the the maximum amount of backups to keep I have backups that are relevant because a player was online at the time, and they won't be deleted after a set amount of days but I get to keep a healthy amount of backups available in case something goes wrong.

Please let me know what you think. Thanks

**Comment by mbround18** [comment]
@JohanVlugt right now we support `AUTO_BACKUP_DAYS_TO_LIVE` which gives you a x amount of backups if you do 50 days at 2 backups a day you get 100 backups :)

**Reply by JohanVlugt** [reply]
The point of my situation is that I want the backups to be not time sensitive. In the current situation if I shut down my server for 100 days for example, all backups will be deleted when I start it up again.

With the maximum amount of backups to keep I don't have to set the `days to live` variable and I get to keep my backups if I don't use my server for 100 days. I hope it makes sense.
